examples/osggeometry/osggeometry.cpp

```diff
@@ -510,8 +510,10 @@ osg::Node* createScene()
         geode->addDrawable(polyGeom);
     }
 
+    geode->getOrCreateStateSet()->setMode(GL_LIGHTING,
+                    osg::StateAttribute::OFF | osg::StateAttribute::PROTECTED);
     return geode;
-}
+} //createScene()
```

Comment:
filnet, thanks for looking at this!  Happy to change it if you wish.  I have personally found this sort of comment helpful, especially when using `less` or another viewer that doesn't have jump-to-matching-brace.